docs/tutorials/12_leveraging_qiskit_runtime.ipynb

```diff
@@ -0,0 +1,695 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "aff59b78",
+   "metadata": {},
+   "source": [
+    "# The QAOA Runtime\n",
```

Comment:
Could you make the file name and title consistent, for example, "Leveraging QAOA runtime" and "12_leveraging_qaoa_runtime.ipynb"? Just renaming "12_qaoa_runtime.ipynb" might be sufficient.

Reply:
Done here: 25945db